### DISCUSSION TITLE
citrix-workspace autostart on boot

### DISCUSSION THREAD

**Discussion by miquecg** [Homebrew/discussions#4098] [Casks] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Hi,

I was wondering if there's a chance that an upgrade of [citrix-workspace cask](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/citrix-workspace.rb) would overwrite changes in plist files.

Somehow, app behaviour defaults to launch on login, which is annoying if you don't need to run it everyday. Lots of users complain about this long ago and apparently the only way to avoid this requires user manual intervention:

> In /Library/LaunchAgents, I had to delete com.citrix.ReceiverHelper.plist and for com.citrix.ServiceRecords.plist and com.citrix.safariadapter.plist open the files and set the value "RunAtLoad" to "false".
>
> Can't really make sense of it but it worked. I didn't touch the files in /Library/LaunchDaemons/.

https://discussions.citrix.com/topic/400690-macos-prevent-workspacereceiver-from-starting-at-loginboot/

Apart from that, would it be possible for the cask itself to allow users to opt-out of this behaviour? I'm not familiar enough with brew, but from what I know about other package managers, some degree of patching is possible. That would be awesome 🙂 

### Relevant casks

citrix-workspace

**Comment by SMillerDev** [comment, marked as answer]
No, casks use the upstream installer as-is.

**Reply by miquecg** [reply]
Ok, so now the other question. Will those files be put in place again when next update comes?

**Reply by SMillerDev** [reply]
If the installer installs them, yes

**Reply by miquecg** [reply]
Thanks.
